timed/reports/filters.py

```diff
@@ -50,8 +49,7 @@ def filter_cost_center(self, queryset, name, value):
         * When viewing the statistic over tasks, only the tasks
           are filtered
         """
-
-        # TODO Discuss: Is this the desired behaviour by our users?
+        # TODO: Discuss Is this the desired behaviour by our users?  # noqa: TD002,TD003
```

Comment:
the todo linter wants us to add a responsible person's username and provide a link to an issue

we might also just consider satisfying it where applicable.

in this concrete instance the todo looks like left-overs from pre-merge times and not really like an actual todo tho

Reply:
I think that's actually a good idea, as otherwise, TODOs do get lost over time. Just the other day I was stumbling over a TODO from over 12 years ago :grin: 

BUT: Given that Timed-backend is due to be merged with the frontend repo, the issue links will likely all change, thus I'd suggest disabling this rule for now

Reply:
they have been disabled